client/app/components/Container.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+
+import { StyleSheet, View } from 'react-native';
+
+const Container = (props) => {
```

Comment:
Same as above - not necessary with native-base but if I were doing it, I'd make it a proper react component (e.g. `class Container extends React.Component`)